### DISCUSSION TITLE
[feature request] Add CONTRIBUTING.md template like CODE_OF_CONDUCT.md template

### DISCUSSION THREAD

**Discussion by rickstaa** [community/community#24401] [Discussions]
It would be nice to have templates for the `CONTRIBUTING.md` file similar to the `LICENSE.md`, and `CODE_OF_CONDUCT.md` would decrease the project setup time. I think something like [this](https://gist.github.com/briandk/3d2e8b3ec8daf5a27a62) would be sufficient.

**Comment by DeveloppeurPascal** [comment]
Hi. You can add yours CONTRIBUTING.md file in the .github repository for all of your repositories. Just copy one you like from an other project and adapt it.

**Reply by rickstaa** [reply]
@DeveloppeurPascal Thanks for your answer. That is indeed what I'm currently doing. I, however, think it would be an improvement if there was a simple CONTRIBUTING template that I could use for small repositories for which I do not want to add custom contributing guidelines.
